src/fabric-1.19.2/templates/gui/gui_msg_button.java.ftl

```diff
@@ -65,4 +66,4 @@ public class ${name}ButtonMessage extends FriendlyByteBuf {
 		});
 	}
 }
-<#-- @formatter:on -->
\ No newline at end of file
+<#-- @formatter:on -->
```

Comment:
Please don't always change them. It shouldn't be chabged in theory.

Reply:
What shouldn't always be changed? This is how it is done in the official generators.

Reply:
If you mean the formatter line then I'm pretty sure that only appears as being changed due to the difference in the amount of lines, all I did was add one line to the template.

Reply:
Ok, this is strange because it shouldn't appear all the time (as otherwise all others lines would also be changed).